frontend/packages/ceph-storage-plugin/integration-tests/views/installFlow.view.ts

```diff
@@ -130,78 +157,39 @@ export class InstallCluster {
     // for 5 Succeeded status in row to be sure we have operator is
     // installed properly.
-    await waitFor(ocsOperatorStatus, 'Succeeded', 5);
-  }
-
-  async installOperator() {
-    await browser.wait(until.and(crudView.untilNoLoadersPresent), 100000);
-    await customNamespaceRadio.click();
-    await browser.wait(until.and(crudView.untilNoLoadersPresent));
-    await browser.wait(until.visibilityOf(dropdownForNamespace));
-    await dropdownForNamespace.click();
-    const nsTag = selectNamespace(this.namespace);
-    await nsTag.click();
-    await primaryButton.click();
-    await browser.wait(until.and(crudView.untilNoLoadersPresent));
-    await browser.wait(until.visibilityOf(crudView.textFilter));
-    await crudView.textFilter.sendKeys(OCS_OPERATOR_NAME);
+    await waitFor(currentSelectors.ocsOperatorStatus, 'Succeeded', 5);
   }
 
   async createStorageCluster() {
-    await goToInstalledOperators();
-    await browser.wait(until.visibilityOf(searchInputOperators));
-    await searchInputOperators.sendKeys(OCS_OPERATOR_NAME);
-    await click(ocsOperator);
-    const storageClusterIndex = await getOperatorHubCardIndex('Storage Cluster');
-    const link = $(`article:nth-child(${storageClusterIndex}) a`);
-    // Operators page does not directly show tiles so refresh until it shows
-    await refreshIfNotVisible(link, 5);
-    await click(link);
-    await browser.wait(until.and(crudView.untilNoLoadersPresent));
-    // Node list fluctautes
-    await browser.sleep(20 * SECOND);
-    const nodes = await selectWorkerRows();
-    // Fluctating
-    await browser.sleep(5 * SECOND);
-    await click(primaryButton);
-    // Let it move to the other page
-    await browser.sleep(1000);
-    await browser.wait(until.visibilityOf(status));
-    await browser.wait(until.visibilityOf(crudView.resourceTitle));
-    return nodes;
-  }
-
-  async createNamespace() {
-    await browser.get(`${appHost}/k8s/cluster/namespaces`);
-    await crudView.isLoaded();
-    const exists = await crudView.rowForName(this.namespace).isPresent();
-    if (!exists) {
-      await crudView.createYAMLButton.click();
-      await browser.wait(until.presenceOf($('.modal-body__field')));
-      await $$('.modal-body__field')
-        .get(0)
-        .$('input')
-        .sendKeys(NS);
-      await $$('.modal-body__field')
-        .get(1)
-        .$('input')
-        .sendKeys(label);
-      await $('.modal-content')
-        .$('#confirm-action')
-        .click();
-      await browser.wait(until.urlContains(`/${this.namespace}`), MINUTE);
+    await click(currentSelectors.ocsOperator);
+    // In fresh clusters APIs are not shown (Last seen in OCP 4.3)
+    try {
+      await browser.wait(until.visibilityOf(currentSelectors.createLink), 10 * SECOND);
+    } catch {
+      await refreshIfNotVisible(currentSelectors.createLink, 5);
     }
+    const storageClusterLink = await currentSelectors.getStorageClusterLink();
```

Comment:
Can you point me to line where you have created `openshift-storage` namespace before installing ocs-operator

Reply:
One doesn't have to create the namespace to install ocs-operator. Operators have the capability to create names by themselves.

Reply:
@umangachapagain Then why do we provide the namespace creation as a prerequisite step, while installing `ocs-operator`